### PR TITLE
loot tables and entities

### PR DESCRIPTION
**Pull request by eristhea**
just general poking around and mapping

### CODE REVIEW

**Comment by EnnuiL** [comment]
hm, it appears that you forgot to undraft this PR

File: mappings/net/minecraft/entity/Tameable.mapping [hunk 1]
```diff
@@ -1,3 +1,4 @@
 CLASS net/minecraft/unmapped/C_lkirduch net/minecraft/entity/Tameable
+	METHOD m_bllryzha getEntityView ()Lnet/minecraft/unmapped/C_nyckmiua;
```
Comment (EnnuiL) [inline mappings/net/minecraft/entity/Tameable.mapping:2]: get entity view from who? i unfortunately couldn't confirm it myself when i strumbled on it, but i suspect it is the owner's entity view

**Review by EnnuiL — CHANGES_REQUESTED** [review]
other than the 1.20-pre5 merger fail and the following poke, this pr looks good!

File: mappings/net/minecraft/entity/Tameable.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/entity/Tameable.mapping:2]: good catch, I'll double check and specify

Comment (eristhea) [reply mappings/net/minecraft/entity/Tameable.mapping:2]: purely based on the implementation of `getOwner`, I think the entity view belongs to the original entity
despite not being able to locate implementations

Comment (EnnuiL) [reply mappings/net/minecraft/entity/Tameable.mapping:2]: ah i see

**Review by EnnuiL — APPROVED** [review]

**Comment by EnnuiL** [comment]
aw, it'll miss 1.20-pre5 merge

**Review by supersaiyansubtlety — APPROVED** [review]

**Comment by EnnuiL** [comment]
oh nvm

**Pull request merged**